docs/content/contribution-guidelines/contributing-inc.md

```diff
@@ -33,6 +33,27 @@ Your assistance in improving documentation is highly valued, regardless of your
 
 To claim an issue that you are interested in, kindly leave a comment on the issue and request the maintainers to assign it to you.
 
+#### GitHub Slash Commands
+
+KubeStellar uses Prow and GitHub bots to help manage issues and pull requests through slash commands. These commands should be written as comments on their own line:
+
+**Issue Management Commands:**
+- `/assign @username` - Assign an issue to a specific user
+- `/unassign @username` - Remove assignment from a user  
+- `/assign` - Assign the issue to yourself
+- `/unassign` - Remove your assignment
+- `/good-first-issue` - Add the "good first issue" label
```

Comment:
This also did not work in my test at https://github.com/kubestellar/kubestellar/issues/3186#issuecomment-3156328485